src/components/CreateListing/CreateListing.tsx

```diff
@@ -14,7 +14,7 @@ import { Button, SIZE } from "~/components/Button/Button";
 import { TSSDKContext } from "~/contexts/tssdk.context";
 import { env } from "~/env.mjs";
 import type { IOptions } from "~/utils";
-import { COLORS, ICON_SIZES, PAGES, getSwapContract } from "~/utils";
+import { COLORS, ICON_SIZES, PAGES, SWAP_TAG, getSwapContract } from "~/utils";
```

Comment:
I've never seen an import start with `~`, is this the same meaning as a normal path, meaning is this pointing to your `home` folder 🤔 ?

Reply:
Yes! This is a Typescript config, check the following on `tsconfig.json`
```
    "paths": {
      "~/*": ["./src/*"]
    },
```
I think NextJS sets this up